src/Data/List/Relation/Binary/Suffix/Homogeneous/Properties.agda

```diff
@@ -45,3 +45,20 @@ isDecPartialOrder dpo = record
   ; _≟_            = Pointwise.decidable DPO._≟_
   ; _≤?_           = suffix? DPO._≤?_
   } where module DPO = IsDecPartialOrder dpo
+
+
+module _ {A : Set a} where
+
+  open import Data.List.Base using (List; []; _∷_)
+  open import Data.List.Properties using (++-monoid)
+  open import Algebra.Properties.Monoid.Divisibility (++-monoid A)
+  open import Relation.Binary.PropositionalEquality using (_≡_; refl; cong)
+  open import Relation.Binary using (_⇒_)
```

Comment:
These imports should live at the top with `Algebra.Properties.Monoid.Divisibility` renamed as `MonoidDivisibility` and opened here?